deploy/rbac-permissions-operator-config/05-osd-readers.ClusterRole.yaml

```diff
@@ -87,3 +66,281 @@ rules:
   verbs:
   - get
   - list
+### END
+### Start - Allow read permissions to more resources
+# https://issues.redhat.com/browse/OSD-5537
+- apiGroups:
+  - admissionregistration.k8s.io
+  resources:
+  - '*'
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - apiextensions.k8s.io
+  resources:
+  - customresourcedefinitions
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - apiregistration.k8s.io
+  resources:
+  - apiservices
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - authentication.k8s.io
+  resources:
+  - tokenreviews
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - authorization.k8s.io
+  resources:
+  - selfsubjectaccessreviews
+  - subjectaccessreviews
+  - selfsubjectrulesreviews
```

Comment:
What is the use case for granting get,list,watch on these?  Given read access is not granted with 'view' clusterrole have we looked sufficiently into _why_?

Reply:
I was following the [google sheet](https://docs.google.com/spreadsheets/d/11Ldw9DmxP8nwvIF7oSMyum_NJNXK9wy9aB_D5CxmNJA/edit#gid=15405060) we used for review of srep backplane roles. 

I was also not aware of the view _clusterrole_, good to know. I can't think of a use case in mind so will remove it.

Reply:
done